docs/sftt/sftt-api-spec.md

```diff
@@ -1822,6 +1822,40 @@ If the request body is malformed.
 
 If the id specified is invalid. An invalid id is a non-existent id or the id of a deleted site.
 
+### Edit a Site (Admin Only)
+
+`POST api/v1/protected/sites/:site_id/edit`
+
+Used to edit the features of a site. This is done by querying the appropriate site and setting the given features. Every field can be `NULL`.
```

Comment:
⛏️ Might be good to note that a null means the feature won't be updated.

Reply:
Oh I think putting `null` sets that column to `null` and the feature won't be updated only if it's not included in the request body. Should I change the route so that `null` means the feature won't be updated or clarify in the doc that a feature will only remain unchanged if it isn't included in the request body?